src/presentation/webapp/core/pages/manual-sync/ManualSyncPage.tsx

```diff
@@ -303,8 +302,12 @@ const ManualSyncPage: React.FC = () => {
                 />
             )}
 
-            {!!syncReport && (
-                <SyncSummary response={syncReport} onClose={() => setSyncReport(null)} />
+            {!!syncReport && destinationInstance && (
+                <SyncSummary
+                    response={syncReport}
+                    instance={destinationInstance.id}
```

Comment:
We might want name instead of id:

```suggestion
                    instance={destinationInstance.name}
```